deploy_stable.bat

```diff
@@ -0,0 +1,20 @@
+@echo on
+set JAVA_HOME="C:\Users\Public\wpilib\2024\jdk
+setlocal
+
+for /f "tokens=2 delims=:" %%a in ('"ipconfig | findstr IPv4"') do set ip=%%a
+set ip=%ip:~1%
+
+rem using ipinfo.io API to get region
+for /f "tokens=3 delims=\": %%i in ('curl -s ipinfo.io/%ip% ^| findstr /C:"city"') do set city=%%i
```

Comment:
You won't have access to the wider internet at a competition.

Reply:
If necessary we can connect to a hotspot (we've done this before) or just deploy manually

Reply:
@Irishumanoid  This won't work even if you're connected to a hotspot because then you wouldn't be connected to the robot's wifi. You'd need to be connected over a wire. At that point, people will find it too complicated to use this script. This script should be: "I click, it goes." The entire point of this script is to be as simple as possible to help the drivers. The drivers should not have to get a wire to connect to the roborio and then connect to a hotspot just to deploy the code. You also would take up 2 minutes at least out of the 8 that you have in a competition. It's also completely unrealistic to expect there to be mobile data to deploy. You're typically buried deep inside a building with unreliable data transfer.

Reply:
It defaults to deploying to main now, so it shouldn't break either way. Is it okay to merge now?

Reply:
The code is untested, so no. Only the simpler version was tested.